src/patterns/ec2-app/base.ts

```diff
@@ -470,6 +511,45 @@ export class GuEc2App extends Construct {
       open: access.scope === AccessScope.PUBLIC,
     });
 
+    if (props.googleAuth?.enabled) {
+      const configPrefix = `${scope.stage}/${scope.stack}/${app}`;
+      const clientIdPath = props.googleAuth.clientIdPath ?? `/${configPrefix}/googleClientID`;
```

Comment:
I think [destructuring with default values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value) would be clearer here, as we can group things together more easily:

```ts
const { googleAuth } = props;
const {
  clientIdPath = `/${configPrefix}/googleClientID`,
  clientSecretPath = `${configPrefix}/clientSecret`
} = googleAuth;
```